packages/docs/src/page-configs/getting-started/configuration-guide/code-examples/icons.ts

```diff
@@ -1,6 +1,6 @@
 export const iconsInstall = `
 yarn add material-design-icons-iconfont -D
-// {{ $t('all.code.or') }}
+// {{ $t('configurationGuide.icons.code.or') }}
```

Comment:
Any reason not to use "all.code.or"?

Reply:
Thanks @asvae, I try organize translation around the `page-config` which the majority of doc info put together, I think it would be easier to find and organize. And put them under `all.code` is harder to seen the context, and eventually harder to find.

Reply:
I have reread the posts, is it because there are few changes to make, so I can put them all inside `all.code` category?

Reply:
Checked code and things look good :).

Using all is not mission critical.